packages/app/test/cypress/integration/20-basic-features/access-to-page.spec.ts

```diff
@@ -143,32 +143,56 @@ context('Access to Template Editing Mode', () => {
 
 
   it('Access to Template Editor mode for only child pages successfully', () => {
-     cy.visit('/Sandbox/Bootstrap4', {  });
-     cy.get('#grw-subnav-container').within(() => {
-       cy.getByTestid('open-page-item-control-btn').click();
-       cy.getByTestid('open-page-template-modal-btn').click();
+    cy.visit('/Sandbox/Bootstrap4', {  });
+
+    cy.get('.grw-skelton').should('exist');
+    cy.get('.grw-skelton').should('not.exist');
+
+    cy.get('#grw-subnav-container').within(() => {
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(300);
```

Comment:
1回のクリックで2回レンダーされているため、時々、`cy.click() failed because this element is detached from the DOM.` が出ていた。

これはおそらく react strict mode による仕様としての影響と思われ、Production ではおそらく再現しない。
詳細：https://ja.reactjs.org/docs/strict-mode.html#detecting-unexpected-side-effects

![re-redner-itme-control-btn](https://user-images.githubusercontent.com/35527421/202089040-bf5c3c66-b509-46a5-a297-a5d69b5388e0.gif)